model/src/main/kotlin/Project.kt

```diff
@@ -155,7 +155,25 @@ data class Project(
     /**
      * A comparison function to sort projects by their identifier.
      */
-    override fun compareTo(other: Project) = id.compareTo(other.id)
+    override fun compareTo(other: Project): Int {
```

Comment:
Are you aware of `compareToBuilder` like `org.apache.commons.lang3.builder.CompareToBuilder` or rather did you consider using one?

Reply:
I was only aware of Guava's *ComparisonChain*, but didn't want to introduce a (big) dependency for only this one (small) use anyway.

Reply:
Did you consider using `compareBy` instead?
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.comparisons/compare-by.html

Kotlin also provides a few more helpers for writing comparators that might be handy here:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-comparator/